lib/lobbyist/v2/customer_suggestion.rb

```diff
@@ -9,6 +9,9 @@ def self.list(company_id, params = {})
         create_collection_from_response(get("/v2/companies/#{company_id}/customer_suggestions.json", params))
       end
 
+      def self.create(company_id, email_key, params = {})
+        create_from_response(post("/v2/companies/#{company_id}/customer_suggestions.json", {'customer_suggestion' => params, 'email_key' => email_key}))
```

Comment:
@sreevs we want this endpoint to be more generic, when we create a customer_suggestion, we dont need an email key. I would suggest do the following
1. Create a new endpoint /v2/companies/:id/unhappy-feedback as POST
2. Point your lobbyist to use that.